core/services/relay/evm/functions/contract_transmitter.go

```diff
@@ -121,8 +116,14 @@ func (oc *contractTransmitter) Transmit(ctx context.Context, reportCtx ocrtypes.
 		oc.lggr.Warnw("failed to generate tx metadata for report", "err", err)
 	}
 
-	destinationContract := oc.contractAddress
-	if oc.contractVersion == 1 {
+	var destinationContract common.Address
+	if oc.contractVersion == 0 {
```

Comment:
Shouldn't this be `oc.contractVersion == 1`? 1 is mainnet contracts and 0 is current beta contracts, right?

Reply:
Nvm.  Commented before I read all the code